src/shared/pkcs11-util.c

```diff
@@ -212,13 +212,17 @@ int pkcs11_token_login(
                                                "Failed to log into security token '%s': %s", token_label, p11_kit_strerror(rv));
 
                 log_info("Successfully logged into security token '%s' via protected authentication path.", token_label);
-                *ret_used_pin = NULL;
+                if (ret_used_pin) {
+                    *ret_used_pin = NULL;
+                }
```

Comment:
Coding style nitpicks: 8ch indenting please, like the rest of the file.

No {} around single line if blocks.

See https://systemd.io/CODING_STYLE/#formatting

Reply:
Corrected